src/containers/App.js

```diff
@@ -58,10 +70,34 @@ class App extends Component {
       selectedEmoji,
       // emojiQuestion,
       mcQuestions,
+      fields,
       dispatch
     } = this.props;
+
+    log(['We got fields!', fields]);
     return (
-      <div className="App">
+      <div className="App" ref={(node) => { this.root = node; }}>
+
+        <h1 className={'intro-title'}>
+          {
+            fieldValue(
+              fields,
+              'elc-text-title',
+              'Word to the President'
+            )
+          }
+        </h1>
+
+        <div className={'intro-blurb'}>
+          {
+            fieldValue(
+              fields,
+              'elc-text-intro-blurb',
+              'The election is over. It’s time to plan for a new administration.'
+            )
+          }
```

Comment:
you have to do the dangerouslySetInnerHTML trick: https://facebook.github.io/react/docs/dom-elements.html#dangerouslysetinnerhtml.

```js
<div dangerouslySetInnerHTML={{ __html: fieldValue(...) }} />
```

Probably best to hide that in a component that just takes in the field value args